README.md

```diff
@@ -1,6 +1,11 @@
 Python WVA Library
 ==================
 
+[![Build Status](https://travis-ci.org/digidotcom/python-wvalib.svg?branch=master)](https://travis-ci.org/digidotcom/python-wvalib)
+[![Latest Version](https://pypip.in/version/wva/badge.svg)](https://pypi.python.org/pypi/wva/)
+[![Supported Python versions](https://pypip.in/py_versions/wva/badge.svg)](https://pypi.python.org/pypi/wva/)
+[![License](https://pypip.in/license/devicecloud/badge.svg)](https://pypi.python.org/pypi/devicecloud/)
```

Comment:
I realize they're the same license (MPL 2.0) but should these links be `wva` and not `devicecloud`?